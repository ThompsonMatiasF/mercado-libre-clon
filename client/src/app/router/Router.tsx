import React, { FC } from 'react';
import { Router as BrowserRouter } from 'react-router-dom';
import { history } from '.';

const Router: FC = ({ children }) => (
  <BrowserRouter history={history}>
    {children}
  </BrowserRouter>
);

export default Router;
