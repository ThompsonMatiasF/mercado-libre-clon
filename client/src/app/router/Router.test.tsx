import { render } from '@testing-library/react'
import Router from './Router'

describe('Router.tsx', () => {
  it('Should match snapshot', () => {
    const { container } = render(<Router>children</Router>)

    expect(container).toMatchSnapshot()
  })
})

