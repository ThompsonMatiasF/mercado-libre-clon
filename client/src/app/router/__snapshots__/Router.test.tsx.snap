// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router.tsx Should match snapshot 1`] = `
<div>
  children
</div>
`;
